pom.xml

```diff
@@ -94,25 +94,19 @@
     <dependencyManagement>
         <dependencies>
             <dependency>
-                <groupId>javax.enterprise</groupId>
-                <artifactId>cdi-api</artifactId>
-                <version>1.2</version>
-                <exclusions>
-                    <exclusion>
-                        <groupId>javax.el</groupId>
-                        <artifactId>javax.el-api</artifactId>
-                    </exclusion>
-                </exclusions>
+                <groupId>jakarta.enterprise</groupId>
```

Comment:
Should we not exclude EL also in the Jakarta artifact?  (General rule from MicroProfile)

Reply:
my bad for doing this at 10pm :) thanks